earthquakes.py

```diff
@@ -20,38 +14,32 @@ def get_data():
             "endtime": "2018-10-11",
             "orderby": "time-asc"}
     )
-
-    # The response we get back is an object with several fields.
-    # The actual contents we care about are in its text field:
     text = response.text
-    # To understand the structure of this text, you may want to save it
-    # to a file and open it in VS Code or a browser.
-    # See the README file for more information.
-    ...
-
-    # We need to interpret the text to get values that we can work with.
-    # What format is the text in? How can we load the values?
-    return ...
+    data = json.loads(text)
+    return data["features"]
 
 def count_earthquakes(data):
     """Get the total number of earthquakes in the response."""
-    return ...
+    return len(data)
 
 
 def get_magnitude(earthquake):
     """Retrive the magnitude of an earthquake item."""
-    return ...
+    return earthquake["properties"]["mag"]
 
 
 def get_location(earthquake):
     """Retrieve the latitude and longitude of an earthquake item."""
-    # There are three coordinates, but we don't care about the third (altitude)
-    return ...
+    return earthquake["geometry"]["coordinates"][0:2]
 
 
 def get_maximum(data):
     """Get the magnitude and location of the strongest earthquake in the data."""
-    ...
+    mags = [get_magnitude(quake) for quake in data]
+    max_value = max(mags)
+    index = mags.index(max_value)
+    return [max_value, get_location(data[index])]
```

Comment:
Since you know you will always be returning two items (the magnitude and location), you can also return a tuple, implicitly or explicitly:
```python
return (max_value, get_location(data[index]))
# or
return max_value, get_location(data[index])
```
Returning a list is by no means wrong, though.